### PR TITLE
Suthesh/multiplier page

### PR DESCRIPTION
**Pull request by suthesan-deriv**
# Description

Added forex and synthetic indices tab in multiplier page

Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/trade-types/multiplier/_available-trades.js [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+import { Header } from 'components/elements'
+import device from 'themes/device'
+//SVG
+import ForexIcon from 'images/svg/market-forex.svg'
+import SyntheticIcon from 'images/svg/market-synthetic-indices.svg'
+
+const StyledHeader = styled(Header)`
+    @media ${device.tabletL} {
+        max-width: 35.75rem;
+        font-size: 4rem;
```
Comment (AbdallaMourad) [inline src/pages/trade-types/multiplier/_available-trades.js:15]: should we use `px` here?

File: src/pages/trade-types/multiplier/_margin.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+`
+
+const StyledTitle = styled(Text)`
+    text-align: left;
+    font-weight: bold;
+`
+
+const Margin = () => {
+    return (
+        <SectionContainer padding="4rem 0 8rem 0">
```
Comment (AbdallaMourad) [inline src/pages/trade-types/multiplier/_margin.js:65]:
```suggestion
        <SectionContainer padding="4rem 0 8rem">
```

File: src/pages/trade-types/multiplier/_synthetic-indices.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+import device from 'themes/device'
+
+const Col = styled(Flex)`
+    max-width: 13.2rem;
+
+    @media ${device.tabletL} {
+        max-width: 12rem;
+    }
+`
+const Row = styled(Flex)``
```
Comment (AbdallaMourad) [inline src/pages/trade-types/multiplier/_synthetic-indices.js:18]: Can we omit this and use `Flex` component instead

File: src/pages/trade-types/multiplier/_margin.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    text-align: center;
+
+    @media ${device.tabletL} {
+        font-weight: 600;
+    }
+`
+
+const StyledTitle = styled(Text)`
+    text-align: left;
+    font-weight: bold;
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_margin.js:60]: Can omit this as `text-align` defaults to the left. We can pass the `weight` as a prop to `Text`.

File: src/pages/trade-types/multiplier/_margin.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        }
+    }
+
+    @media ${device.mobileL} {
+        grid-template-columns: repeat(2, 1fr);
+    }
+`
+const StyledText = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 2rem;
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_margin.js:46]: Should this be in `px`?

File: src/pages/trade-types/multiplier/_margin.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+        text-align: left;
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_margin.js:47]: Can omit as it defaults to left.

File: src/pages/trade-types/multiplier/_margin.js [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+    @media ${device.tabletL} {
+        grid-template-columns: repeat(3, 1fr);
+
+        img {
+            width: 16px;
+            height: 16px;
+            margin-right: 4px;
+        }
+        ${Text} {
+            font-size: 1.5rem;
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_margin.js:35]: Should this be in `px`?

File: src/pages/trade-types/multiplier/_available-trades.js [hunk 8]
```diff
@@ -0,0 +1,246 @@
...
+        h4 {
+            color: ${(props) =>
+        props.active_tab === props.name ? 'var(--color-red)' : 'var(--color-black-3)'};
+            opacity: ${(props) => (props.active_tab === props.name ? '1' : '0.56')};
+        }
+        @media ${device.tabletL} {
+            width: 100%;
+            height: 100%;
+            justify-content: flex-start;
+            padding: 10px 44px 0 0;
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_available-trades.js:85]:
```suggestion
            padding: 10px 44px 0;
```

File: src/pages/trade-types/multiplier/_available-trades.js [hunk 9]
```diff
@@ -0,0 +1,246 @@
...
+        border-bottom: none;
+        border-radius: 10px 30px 0 0;
+        background: #f3f3f3;
+        transform: perspective(14px) rotateX(1.4deg);
+        transform-origin: bottom left;
+        box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.05);
+        ${(props) => {
+        if (props.active_tab === props.name)
+            return css`
+                    background-color: #ffffff;
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_available-trades.js:112]:
```suggestion
                    background-color: var(--color-white);
```

File: src/pages/trade-types/multiplier/_available-trades.js [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+        width: 100%;
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: -1rem;
+        left: 0;
+        z-index: -1;
+        border-bottom: none;
+        border-radius: 10px 30px 0 0;
+        background: #f3f3f3;
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_available-trades.js:105]: Can we add this to `variables`?

File: src/pages/trade-types/multiplier/_available-trades.js [hunk 11]
```diff
@@ -0,0 +1,246 @@
...
+                margin-left: 16px;
+            `
+    }}
+`
+
+const ContentWrapper = styled.div`
+    width: 100%;
+    max-width: 99.6rem;
+    display: block;
+    background: #ffffff;
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_available-trades.js:147]: Can use from added `variables`

File: src/pages/trade-types/multiplier/_available-trades.js [hunk 12]
```diff
@@ -0,0 +1,246 @@
...
+    }
+    handleTabChange = (new_tab) => {
+        if (new_tab === this.state.active_tab) return
+        this.setState({ active_tab: new_tab })
+    }
+    render() {
+        const { display_title, Forex, SyntheticIndices } = this.props
+        return (
+            <StyledSection>
+                <StyledHeader size="var(--text-size-header-1)" align="center">
```
Comment (sean-binary) [inline src/pages/trade-types/multiplier/_available-trades.js:208]: `type`? can omit `size` if type is sufficient. We should add `as` for semantics as well.

File: src/pages/trade-types/multiplier/_synthetic-indices.js [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+    text-align: center;
+
+    @media ${device.tabletL} {
+        font-weight: 600;
+    }
+`
+
+const StyledTitle = styled(Text)`
+    text-align: left;
+    font-weight: bold;
```
[the remark this answers is not in the document]
Comment (suthesan-deriv) [reply src/pages/trade-types/multiplier/_synthetic-indices.js:69]: Actually it should have been set to center by default, then made to align left on mobile...fixed it man 👍

File: src/pages/trade-types/multiplier/_available-trades.js (hunk 12 above)
Comment (suthesan-deriv) [reply src/pages/trade-types/multiplier/_available-trades.js]: Looks like type can't fit in here...but we can add `as` though 👌

**Review by sean-binary — APPROVED** [review]

**Review by AbdallaMourad — APPROVED** [review]

**Review by yashim-deriv — APPROVED** [review]

**Review by mustofa-binary — CHANGES_REQUESTED** [review]
please help to rebase

**Pull request merged**